services/src/main/java/org/fao/geonet/api/pages/AccessExpression.java

```diff
@@ -0,0 +1,15 @@
+package org.fao.geonet.api.pages;
+
+import java.util.List;
+
+public class AccessExpression {
```

Comment:
Access Expression is supposed to be a generic field that can be used to store the expression based on the page status.
In this case it is a page expression based on the groups status.  If a new status is created which requires an expression, it will most likely not have groups field.

So it is recommended that you rename this to GroupsAccessExpression

It is also recommended that you do like the `site` and `users` api folder and put this class in a `model` folder.

Reply:
Also add some basic documentation for this class as well as include the license header.

Reply:
I added the license. Also renamed the class.